DOCS-9716 GovCloud (#186)
mulesoft/docs-api-community-manager: modules/ROOT/pages/connectivity-govcloud.adoc

> = Connectivity with Anypoint GovCloud

Reviewer: Per the PMM product tiering and naming source of truth, this is not the product name, regardless of how people might speak of it:

Official name | MuleSoft Government Cloud | Full name to avoid confusion
Never refer to as... | Government Cloud, GovCloud | Government Cloud can refer to the AWS region within DoD, so we should avoid that conflict.

Additionally, our MuleSoft Term Presentation Reference contains this entry:

**MuleSoft Government Cloud**
Do not refer to as Government Cloud alone.

Reviewer: ```suggestion
= Connectivity with MuleSoft Government Cloud
```
There is no acceptable abbreviation, due to the conflict with the trademarked AWS GovCloud. GLOBAL

Author: Fixed.

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> API Community Manager can be used to create developer communities that surface APIs from Anypoint Platform

Reviewer: passive voice; reword:
```suggestion
Use Anypoint API Community Manager to create developer communities that surface APIs from Anypoint Platform
```
Product Tiering and Hierarchy:
https://docs.google.com/spreadsheets/d/1f6v9cjx84ZAHGeZs36A0clZoQc_ggAe6m6e-NQ9BUuk/edit#gid=71218867

Official name | Anypoint API Community Manager |  
Alternatives | API Community Manager | Never ACM

GLOBAL

Author: Fixed.

> organizations in the MuleSoft Government Cloud (GovCloud) environment. In this deployment model, your developer community runs

Reviewer: NO abbreviation for this product name:
```suggestion
organizations in the MuleSoft Government Cloud environment. In this deployment model, your developer community runs
```

Author: Fixed.

> on the standard API Community Manager product in the commercial cloud environment, and it connects to your

Reviewer: Break up super long sentence:
```suggestion
on the standard API Community Manager product in the commercial cloud environment. <Replace "it" with what "it" refers to>) connects to your
```

Author: Fixed.

> Anypoint GovCloud organization as an external system to retrieve the specific information you want to make available

Reviewer: ```suggestion
MuleSoft Government Cloud organization as an external system to retrieve the specific information you want to make available
```

Author: Fixed.

outside your FedRAMP authorization boundary for your public API community, such as API documentation.

> 

Reviewer: Reread this paragraph ^. It seems awfully wordy and repetitive to me; any way to simplify and not use all those names over and again? Perhaps not, but it's worth a try!

Author: Updated.

> To ensure data security in this model, you have full control of what data in your Anypoint GovCloud organization is

Reviewer: For legal reasons, we are not ever to "ensure" anything with our products but rather "help you to ensure" or "helps ensure" (see style ref.):
```suggestion
To help ensure data security in this model, you have full control of what data in your organization is
```

Author: Fixed.

> accessible for the API Community Manager product in the commercial cloud environment. The GovCloud data control

Reviewer: ```suggestion
accessible to the API Community Manager product in the commercial cloud environment. The MuleSoft Government Cloud data control
```

Author: Fixed.

> documentation page shows how to configure these fine-grain data access controls.

Reviewer: "fine-grain" is idiomatic, specific to photography. Consider replacing with something like "detailed" or "finely detailed."

Reviewer: GLOBALLY replace all GovCloud references with proper naming.
I will stop marking this so as to save us both time; please search and replace.

Author: Fixed.

Additionally, for compliance with FedRAMP regulations on handling support cases associated with API Community
Manager when connected to your Anypoint GovCloud organization, any support case that requires access or interaction

> with your Anypoint organization data is handled by the MuleSoft support team following FedRAMP requirements.

Reviewer: This seems awfully verbose. Can we reduce it to...:

Additionally, any support case that requires access or interaction with your organization data is handled by the MuleSoft support team following FedRAMP requirements.

Reviewer: GLOBAL and per term presentation list, never reduce "Anypoint Platform" to "Anypoint" alone.

Author: Fixed.

> Cases not requiring access to your Anypoint GovCloud data, such as look and feel, branding, CMS articles, and

Reviewer: ```suggestion
Support cases that do not require access to your data (such as those related to look and feel, branding, CMS articles, and
```

Author: Fixed.

> forums on your public developer community, are handled through API Community Manager standard support processes.

Reviewer: ```suggestion
forums on your public developer community) are handled through API Community Manager standard support processes.
```

Author: Fixed.

== See Also

> * xref:govcloud-data-control.adoc[GovCloud Data Control]

Reviewer: rename

Author: Fixed.
